src/components/ImportFilterDialog.tsx

```diff
@@ -47,27 +70,311 @@ export const ImportFilterDialog: React.FC<ImportFilterDialogProps> = ({
+        console.log('Loading github filter:', source)
+        return loadFilter({
+            ...source,
+        })
+            .catch((error) => {
+                setImportError(error.message)
+            })
+            .then((filter) => {
+                if (filter) {
+                    if (filterNameOverride !== '') {
+                        filter.name = filterNameOverride
+                    }
+                    addImportedModularFilter(filter)
+                    setActiveFilterId(filter.id)
+                    handleClose()
+                }
+                return filter
+            })
+    }
+
+    const [loading, setLoading] = useState(false)
+
+    const [showURLImportOptions, setShowURLImportOptions] = useState(false)
+
     return (
-        <Dialog fullWidth open={open} onClose={handleClose}>
-            <DialogTitle>Import Filter</DialogTitle>
+        <Dialog fullWidth maxWidth="xl" open={open} onClose={handleClose}>
             <DialogContent>
+                <Grid2
+                    container
+                    spacing={2}
+                    sx={{ mt: 2, justifyContent: 'center' }}
+                >
+                    <Grid2 size={3}>
+                        <Typography
+                            variant="h6"
+                            fontSize="36px"
+                            component="div"
+                            sx={{ color: 'text.primary', fontSize: '24px' }}
+                        >
+                            What is this Website?
+                        </Typography>
+                        <Typography
+                            variant="body2"
+                            sx={{ color: '#CCCCCC', fontSize: '24px' }}
+                        >
+                            <p>
+                                FilterScape.xyz makes it easy to customize how
+                                your loot-filters plugin filters look and feel.
+                            </p>
+                            <p>
+                                It provides previews for your item labels,
+                                groups configurations by item type, and more.
+                            </p>
+                        </Typography>
+                    </Grid2>
+
+                    <Grid2 size={1} />
+
+                    <Grid2 size={3}>
+                        <Typography
+                            variant="h6"
+                            fontSize="36px"
+                            component="div"
+                            sx={{ color: 'text.primary', fontSize: '24px' }}
+                        >
+                            What is the Loot Filters Plugin?
+                        </Typography>
+                        <Typography
+                            variant="body2"
+                            sx={{ color: '#CCCCCC', fontSize: '24px' }}
+                        >
+                            <p>
```

Comment:
Might be nice to have a link to [Loot Filters plugin hub page](https://runelite.net/plugin-hub/show/loot-filters) somewhere here. Possibly easier for people to understand the use case.

Maybe `Loot Filters` is a link?